dpcpp/matrix/dense_kernels.dp.cpp

```diff
@@ -205,7 +205,7 @@ void calculate_slice_lengths(size_type num_rows, size_type slice_size,
                              size_type* __restrict__ slice_sets,
                              sycl::nd_item<3> item_ct1)
 {
-    constexpr auto sg_size = cfg;
+    constexpr auto sg_size = KCFG_1D::decode<1>(cfg);
```

Comment:
This looks like it was a bug before?

Reply:
No, the cfg is subgroup list only. it fixes the workgroup size

Reply:
The corresponding code was removed, so this is no longer relevant